mods/mixandmega/scripts.js

```diff
@@ -38,8 +38,8 @@ let BattleScripts = {
 		}
 
 		// Do we have a proper sprite for it?
-		// @ts-ignore
-		if (this.getTemplate(pokemon.canMegaEvo).baseSpecies === pokemon.originalSpecies || isUltraBurst) {
+		// @ts-ignore TypeScript bug: TS doesn't infer that if isUltraBurst is false, pokemon.canMegaEvo must be a string
```

Comment:
I wouldn't call this a TS bug; TS was never designed to handle these sorts of complex relationships. This is just an assertion.